src/services/configurations/general.ts

```diff
@@ -7,7 +7,18 @@ export default (
   httpAuth: HttpInstance
 ): ConfigurationsGeneralService => ({
   async get(options) {
-    return (await client.get(httpAuth, '/general', options)).data;
+    const { data } = await client.get(httpAuth, '/general', {
+      ...options,
+      customResponseKeys: [
+        'data.*',
```

Comment:
The `/*` is unnecessary if there is a name before it  It probably will work but you don't really have to put it

Reply:
I find a little conflicting the object is customResponseKeys.
I would expect that it wants specific keys hence the wildcard to re-enforce it's all keys in this object. 🤔 

It works both ways, but it's up to whatever people want.

Reply:
It does not really matter, but it would have been good if this came up in the first PR. Now we are already inconsistent with ourselfves because the profile service and the data service don't do it like that.

Reply:
I agree that this makes more sense though

Reply:
What is the point of the `*` though as a wildcard if we don't consume it? 🤔 

We would almost never user it in the root of something because of timestamps right? 🤔

Reply:
I also think it looks nicer 👍 Although I also agree the extra work isn't ideal, we all want to move to this "`.*` syntax"?

Reply:
@chad-pelly the localization service needs the wildcard, that is why I added it. 

@wwaaijer-exh if we all want to move to the .* syntax someone should go back through the profile and data service and change it. 

Also then we have the question wether `custom_fields` should still work as it does know  or wether that should just not change `custom_fields` in itself.

Reply:
Can we make this a "questions document question" and/or have a quick call soon(-ish)? I feel like we're not all fully aligned. Having this discussion async and spread out over a day is most likely not going to very efficient 🤔